autoload/ale/elixir.vim

```diff
@@ -0,0 +1,21 @@
+" Author: carakan <carakan@gmail.com>
+" Description: Functions for working with Elixir exec.
+
+call ale#Set('elixir_executable', '')
```

Comment:
It's unusual to have an executable option default to an empty string in the codebase. This option could be confused with an option for executing elixir for linting. You ought to default all executables to some name for something that can be found in `PATH`.

If you need to typically run `mix` instead of `elixir mix`, I recommend renaming this option to something like `g:ale_elixir_mix_elixir_executable`, or something. Then it should be okay for that to be an empty string by default, if `mix` is typically better than `elixir mix`.

Document the behavior of both options.

Reply:
@w0rp you're right I'll delete the elixir option.